nibabel/gifti/gifti.py

```diff
@@ -238,19 +241,38 @@ class GiftiDataArray(xml.XmlSerializable):
     data = np.ndarray
     coordsys = GiftiCoordSystem
     meta = GiftiMetaData
+    """
 
-    def __init__(self, data=None):
+    def __init__(self, data=None,
+                 encoding="GIFTI_ENCODING_B64GZ",
+                 endian=sys.byteorder,
+                 coordsys=None,
+                 ordering="C",
+                 meta=None):
+        """
+        Returns a shell object that cannot be saved.
+        """
```

Comment:
Cool. I have to say, I'm not clear enough on the `GIFTI` format to write these; I've simply been approaching `GIFTI` from the software engineering side.

Sounds like you're up for writing these, @matthew-brett ?